changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** WooCommerce Payments Changelog ***
 
 = 2.8.0 - 2021-xx-xx =
+* Add - Enable customer multi-currency.
```

Comment:
What should we use as the changelog entry?

cc @LevinMedia, @dwainm

Reply:
I've added the changelog entry as: 

```
Add - Allow merchants to add additional currencies to their store, allowing a store’s customers to shop and browse in the currency of their choice.
```

Reply:
I submitted it to the release process, but still open to change.

Reply:
Cool, thanks! I've updated it to match that one in https://github.com/Automattic/woocommerce-payments/pull/2552/commits/a7aefe4f3b3d328e6ce9a3754a328a940c4d8624.